library/systemd/test/yast2/system_service_test.rb

```diff
@@ -102,6 +102,28 @@
     end
   end
 
+  describe ".build" do
+    let(:systemd_service) { instance_double(Yast::SystemdServiceClass::Service) }
+
+    it "returns a systemd service even if it does not exist" do
+      expect(Yast::SystemdService).to receive(:build).with("other")
+        .and_return(systemd_service)
+      system_service = described_class.build("other")
+      expect(system_service.service).to eq(systemd_service)
+    end
+  end
+
+  describe ".build" do
```

Comment:
It seems to me that this `describe` is duplicated.

Reply:
Oh, I messed up things rebasing :) Thanks.